web/src/components/FeedbackModal/FeedbackModal.tsx

```diff
@@ -1,22 +1,28 @@
-import { FeedbackSubmittedDialog } from "@/components/betaBar/FeedbackSubmittedDialog";
-import { RequestFeatureDialog } from "@/components/betaBar/RequestFeatureDialog";
-import { SelectFeedbackDialog } from "@/components/betaBar/SelectFeedbackDialog";
+import { FeedbackSubmittedDialog } from "@/components/FeedbackModal/FeedbackSubmittedDialog";
+import { ReportIssueDialog } from "@/components/FeedbackModal/ReportIssueDialog";
+import { RequestFeatureDialog } from "@/components/FeedbackModal/RequestFeatureDialog";
+import { SelectFeedbackDialog } from "@/components/FeedbackModal/SelectFeedbackDialog";
 import { FeedbackRequestDialogNames } from "@/lib/types/types";
 import { ReactElement, useEffect, useState } from "react";
 
 type Props = {
   handleClose: () => void;
   isOpen: boolean;
+  isBugBar?: boolean;
```

Comment:
it might be nice to have consistent domain language throughout. Everywhere else in the code & content we're using the term "issue" - is there a reason we switch to "bug" when referencing this behavior?

Reply:
we even call the component ReportAnIssueBar already